index.html

```diff
@@ -17,7 +19,7 @@ <h1 class="header__title">Title</h1>
       <h2 class="header__subtitle">Subtitle</h2>
     </header>
 
-    <main>
+    <main class="container">
       <section class="section">
         <p class="section__paragraph">
           Lorem ipsum dolor sit amet consectetur adipisicing elit. Provident
```

Comment:
please remove the lorem ipsum text

Reply:
I have already deleted the text lorem ipsum